specification/containerregistry/Azure.ContainerRegistry/tspconfig.yaml

```diff
@@ -0,0 +1,63 @@
+parameters:
+  "service-dir":
+    default: "sdk/containerregistry"
+
+emit:
+  - "@azure-tools/typespec-autorest"
+
+linter:
+  extends:
+    - "@azure-tools/typespec-azure-rulesets/data-plane"
+
+options:
+  "@azure-tools/typespec-autorest":
+    azure-resource-provider-folder: "data-plane"
+    emitter-output-dir: "{project-root}/.."
+    examples-dir: "{project-root}/examples"
+    output-file: "{azure-resource-provider-folder}/Registry/{version-status}/{version}/containerregistry.json"
+  "@azure-tools/typespec-python":
+    package-dir: "azure-containerregistry"
+    package-name: "azure-containerregistry"
+    package-pprint-name: "Azure Container Registry"
+    namespace: "azure.containerregistry"
+    package-version: "1.2.0"
+    flavor: "azure"
+    generate-test: true
+    generate-sample: true
+  "@azure-tools/typespec-java":
+    emitter-output-dir: "{output-dir}/{service-dir}/azure-containers-containerregistry"
+    package-name: "com.azure.containers.containerregistry"
+    package-version: "1.2.19"
+    namespace: "com.azure.containers.containerregistry"
+    generate-test: true
+    generate-sample: true
+    flavor: "azure"
+  "@azure-tools/typespec-csharp":
+    package-name: "Azure.Containers.ContainerRegistry"
+    package-dir: "Azure.Containers.ContainerRegistry"
```

Comment:
Remove `package-dir` and add `emitter-output-dir: "{output-dir}/{service-dir}/{namespace}"`